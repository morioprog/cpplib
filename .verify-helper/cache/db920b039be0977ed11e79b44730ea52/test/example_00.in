5 7
1 2 3 4 5
3 0 5
2 2 4 100
3 0 3
0 1 3 10
3 2 5
1 2 5 20
3 0 5
