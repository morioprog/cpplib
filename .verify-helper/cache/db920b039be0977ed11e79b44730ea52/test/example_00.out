15
106
119
147
